examples/private_connection/backend.tf

```diff
@@ -0,0 +1,22 @@
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+# terraform {
```

Comment:
Follow other examples' backend.tf https://github.com/forseti-security/terraform-google-forseti/blob/master/examples/install_simple/backend.tf

Reply:
do you mean to have bucket name to be `"my-project-tfstate"`?

Reply:
Yes

Reply:
done